server/routes/cohorts.js

```diff
@@ -0,0 +1,17 @@
+const router = require('express').Router();
+
+const { editCohort, deleteCohort } = require('../controllers');
+
+const { getSpecificCohort } = require('../controllers');
+
+const { getCohortsData } = require('../controllers');
+
```

Comment:
you can write it in one line like this :smiley: 
const { editCohort, deleteCohort,getSpecificCohort,getCohortsData } = require('../controllers');